src/main/java/com/conveyal/gtfs/loader/ReferenceTracker.java

```diff
@@ -136,4 +150,120 @@
         }
         return errors;
     }
+
+
+    /**
+     * Work through each conditionally required check assigned to a table. First check the reference field to confirm
+     * if it meets the conditions whereby the conditional field is required. If the conditional field is required confirm
+     * that a value has been provided, if not, log an error.
+     */
+    public Set<NewGTFSError> checkConditionallyRequiredFields(Table table, Field[] fields, String[] rowData, int lineNumber) {
```

Comment:
Line too long

Reply:
Also, can this method have some more inline comments to better explain the things going on inside?

Reply:
Updated.